evm/gas-service.js

```diff
@@ -47,18 +47,20 @@ function printFailedChainUpdates() {
 
 const feesCache = {};
 
-async function getFeeData(api, chain) {
-    if (!feesCache[chain]) {
+async function getFeeData(api, sourceChain, destinationChain) {
```

Comment:
```suggestion
async function getFeeData(api, referenceSourceChain, destinationChain) {
```
maybe?

Reply:
This function is not aware if it's used with reference chain or real source -> dest. It's an API client and from it's perspective it's just source and dest

Reply:
sure